ui-spacetimechart/src/hooks/useMouseInteractions.ts

```diff
@@ -184,7 +125,5 @@ export function useMouse(
         initialData: panningState.initialData,
       });
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [position]);
-
-  return mouseState;
+  }, [position.x, position.y]);
```

Comment:
Do you have a special reason to use these dependencies instead of `[position]` ?

Reply:
If a new `position` object with the same `x` and `y` values is received (with something like `const newPosition = { ...position };` for instance), the old `[position]` dependencies version triggers the effect, while the new `[position.x, position.y]` does not.